kerastuner/tuners/bayesian.py

```diff
@@ -52,7 +53,7 @@ class BayesianOptimizationOracle(oracle_module.Oracle):
     def __init__(self,
                  objective,
                  max_trials,
-                 num_initial_points=2,
+                 num_initial_points=None,
```

Comment:
This needs to be marked as optional in the docstring, with an explanation of what default value gets used

Reply:
Done